books/free-programming-books-langs.md

```diff
@@ -1031,6 +1031,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [Java Programming](https://en.wikibooks.org/wiki/Java_Programming) - Wikibooks
 * [Java Programming for Kids](https://yfain.github.io/Java4Kids/) - Yakov Fain
 * [Java Projects, Second Edition](https://www.packtpub.com/free-ebooks/java-projects-second-edition) - Peter Verhas (Packt account *required*)
+* [Java Web Development](https://education.launchcode.org/java-web-development/) - Chris Bay, Jim Flores, Carly Langlois, Sally Steuterman (The LaunchCode Foundation)
```

Comment:
As Eric says... 

Although most is text and is called itself "book", it has some videos that cant' replace text. E.g:

https://education.launchcode.org/java-web-development/chapters/orm-part1/background.html

Reply:
Resolved in https://github.com/EbookFoundation/free-programming-books/pull/6540/commits/106eb8583651375a54400e43261e4d2930b5d4df